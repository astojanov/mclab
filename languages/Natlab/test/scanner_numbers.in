2
3.4
5.
.6
7E8
8.9E1
2.E3
.4E5
6..7