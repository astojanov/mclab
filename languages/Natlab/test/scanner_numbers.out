INT_NUMBER 1 1 1 =2 as '2'
LINE_TERMINATOR 1 2 1
FP_NUMBER 2 1 3 =3.4 as '3.4'
LINE_TERMINATOR 2 4 1
FP_NUMBER 3 1 2 =5 as '5.'
LINE_TERMINATOR 3 3 1
FP_NUMBER 4 1 2 =0.6 as '.6'
LINE_TERMINATOR 4 3 1
FP_NUMBER 5 1 3 =7E+8 as '7E8'
LINE_TERMINATOR 5 4 1
FP_NUMBER 6 1 5 =89 as '8.9E1'
LINE_TERMINATOR 6 6 1
FP_NUMBER 7 1 4 =2E+3 as '2.E3'
LINE_TERMINATOR 7 5 1
FP_NUMBER 8 1 4 =4E+4 as '.4E5'
LINE_TERMINATOR 8 5 1
FP_NUMBER 9 1 2 =6 as '6.'
FP_NUMBER 9 3 2 =0.7 as '.7'
LINE_TERMINATOR 9 5 1
INT_NUMBER 10 1 1 =2 as '2'
LINE_TERMINATOR 10 2 1
FP_NUMBER 11 1 2 =2 as '2.'
LINE_TERMINATOR 11 3 1
INT_NUMBER 12 1 3 =2 as '0x2'
LINE_TERMINATOR 12 4 1
IM_INT_NUMBER 13 1 2 =2i as '2i'
LINE_TERMINATOR 13 3 1
IM_FP_NUMBER 14 1 3 =2i as '2.i'
LINE_TERMINATOR 14 4 1
IM_INT_NUMBER 15 1 4 =2i as '0x2i'
LINE_TERMINATOR 15 5 1
